pkg/collateral/control.go

```diff
@@ -183,12 +183,27 @@ func (g *generator) genCommand(cmd *cobra.Command) {
 	if cmd.Runnable() {
 		g.emit("<pre class=\"language-bash\"><code>", html.EscapeString(cmd.UseLine()))
 		g.emit("</code></pre>")
-	}
 
-	if len(cmd.Aliases) > 0 {
-		g.emit("<p>")
-		g.emit(cmd.Aliases...)
-		g.emit("<p>")
+		if len(cmd.Aliases) > 0 {
+			// first word in cmd.Use represents the command that is being aliased
+			word := cmd.Use
+			index := strings.Index(word, " ")
+			if index > 0 {
+				word = word[0:index]
+			}
+
+			g.emit("<div class=\"aliases\">")
+			line := cmd.UseLine()
+			for i, alias := range cmd.Aliases {
+				r := strings.Replace(line, word, alias, 1)
+				if i == 0 {
+					g.emit("<pre class=\"language-bash\"><code>", html.EscapeString(r))
+				} else {
```

Comment:
Do we want a `<br>` or similar line break between aliases for commands with >1 alias?  I am thinking of _proxyconfig.go_ which has two aliases.